lib/common/xml.c

```diff
@@ -645,6 +645,7 @@ create_xml_node(xmlNode * parent, const char *name)
     }
 
     if (parent == NULL) {
+        // libxml2 supports only XML 1.0
```

Comment:
re: commit message, turns out that info is also on the libxml homepage :/ ah well, just had to ask him why `xmlNewDoc` took an argument that had to be "1.0"

Reply:
Good ideas.

As a side note, Daniel Veillard said in 2007 that libxml doesn't and likely never would support anything beyond XML 1.0 (and presumably XPath 1.0). And here we are in 2024. I suspect that when we get XML 3.0 in the year 2050, libxml will still support only 1.0.

Reply:
done